From d3d34949d54c1e4e7777a8cbcd7f0d3d78c8448a Mon Sep 17 00:00:00 2001
From: Brad Geltz <brad.geltz@intel.com>
Date: Wed, 7 Oct 2020 11:20:39 -0700
Subject: [PATCH 1/2] Fix whitespace issues
#
#  Copyright (c) 2015 - 2023, Intel Corporation
#  SPDX-License-Identifier: BSD-3-Clause
#

- Add build files to .gitignore

Signed-off-by: Brad Geltz <brad.geltz@intel.com>
---
 .gitignore                | 3 +++
 setup/Make.IMPI_IOMP_AVX2 | 6 +++---
 setup/Make.IMPI_IOMP_SKX  | 6 +++---
 3 files changed, 9 insertions(+), 6 deletions(-)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..14ba1b2
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,3 @@
+*.o
+bin/
+Makefile
diff --git a/setup/Make.IMPI_IOMP_AVX2 b/setup/Make.IMPI_IOMP_AVX2
index e7e42ca..38b0f73 100644
--- a/setup/Make.IMPI_IOMP_AVX2
+++ b/setup/Make.IMPI_IOMP_AVX2
@@ -87,9 +87,9 @@ BINdir       = $(TOPdir)/bin
 # header files,  MPlib  is defined  to be the name of  the library to be
 # used. The variable MPdir is only used for defining MPinc and MPlib.
 #
-MPdir        = 
-MPinc        = 
-MPlib        = 
+MPdir        =
+MPinc        =
+MPlib        =
 #
 #
 # ----------------------------------------------------------------------
diff --git a/setup/Make.IMPI_IOMP_SKX b/setup/Make.IMPI_IOMP_SKX
index 3213ef7..9325d71 100644
--- a/setup/Make.IMPI_IOMP_SKX
+++ b/setup/Make.IMPI_IOMP_SKX
@@ -87,9 +87,9 @@ BINdir       = $(TOPdir)/bin
 # header files,  MPlib  is defined  to be the name of  the library to be
 # used. The variable MPdir is only used for defining MPinc and MPlib.
 #
-MPdir        = 
-MPinc        = 
-MPlib        = 
+MPdir        =
+MPinc        =
+MPlib        =
 #
 #
 # ----------------------------------------------------------------------
-- 
2.23.0

